frontend/src/components/Spinner/Spinner.tsx

```diff
@@ -5,13 +5,30 @@ import S from './styles';
 
 type Props = {
   text: string;
+  size?: string;
 };
 
-const Spinner = ({ text }: Props) => (
-  <S.Wrapper className="loading-spinner">
-    <Spin indicator={<LoadingOutlined style={{ fontSize: '2.5rem' }} spin />} />
-    <p>{text}</p>
-  </S.Wrapper>
-);
+const Spinner = ({ text, size = 'large' }: Props) => {
+  // Small Spinner
+  if (size === 'small') {
+    return (
+      <S.SmallWrapper className="loading-spinner">
+        <Spin indicator={<LoadingOutlined spin />} size="small" />
+        <p>{text}</p>
+      </S.SmallWrapper>
+    );
+  }
+  // Large Spinner (Default)
+  if (size === 'large') {
+    return (
+      <S.LargeWrapper className="loading-spinner">
+        <Spin indicator={<LoadingOutlined spin />} size="large" />
+        <p>{text}</p>
+      </S.LargeWrapper>
+    );
+  }
+
+  return null;
```

Comment:
I think this is fine (and should never be reachable if we pass typecheck), but might lead to bugs that become hard to find. It might be better to just make the large condition the else condition, so we never have to deal with a backup return.